src/features/AppletSettings/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -0,0 +1,141 @@
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { CheckboxController, EditorController, InputController } from 'components/FormComponents';
+import { StyledBodyLarge, StyledTitleMedium } from 'styles/styledComponents/Typography';
+import theme from 'styles/theme';
+import { Svg } from 'components';
+import { variables } from 'styles/variables';
+
+import { StyledAppletSettingsButton, StyledHeadline } from '../AppletSettings.styles';
+import { defaultValues } from './ReportConfigSetting.const';
+import { reportConfigSchema } from './ReportConfigSetting.schema';
+import {
+  StyledButton,
+  StyledContainer,
+  StyledForm,
+  StyledHeader,
+} from './ReportConfigSetting.styles';
+import { FormValues } from './ReportConfigSetting.types';
+
+export const ReportConfigSetting = () => {
+  const { t } = useTranslation();
+  const isServerConfigured = false;
```

Comment:
Pls, add to do to add server configured functionality when the back-end is ready.